Automattic/wp-calypso
# Help Center: Fix Help Center in checkout and Sybil articles

#### Proposed Changes

- **Fix help center style on the checkout page**: On that page, many CSS rules are missing, so I fixed what was broken.
- **More reliable site id**: Help Center packages now doesn't depend on `window._currentSiteId`, but uses `getSelectedSiteId`. ETK sets its value, and if we're in calypso the value is already set.
- Releases help center in checkout page in `development`, `wpcalypso` and `stage`
- **Sybil articles aware of the section**: `Recommended resources` were offered only having `gutenberg-editor` as context. Now we get the value from `getSectionName`
- Fixed an [unnecessary chain condition
](https://github.com/Automattic/wp-calypso/pull/64425/files#diff-ac4b8e65c1a3ef7336ecca35fe4c1be3628a73a08e045b237ad373f2366d8d9bR220-R221)

#### Testing Instructions



* Checkout the branch
* `yarn start`
* Visit `http://calypso.localhost:3000/checkout/SITE.wordpress.com`
* Check that the help center works correctly



Related to #63339
Fixes #63339
